front/components/workspace/ProviderManagementModal.tsx

```diff
@@ -0,0 +1,129 @@
+import { ContextItem, Modal, SliderToggle } from "@dust-tt/sparkle";
+import type { ModelProviderIdType, WorkspaceType } from "@dust-tt/types";
+import { MODEL_PROVIDER_IDS, SUPPORTED_MODEL_CONFIGS } from "@dust-tt/types";
+import _ from "lodash";
+import { useCallback, useMemo, useState } from "react";
+
+import { MODEL_PROVIDER_LOGOS } from "@app/components/assistant_builder/InstructionScreen";
```

Comment:
It feels weird to import this from `InstructionScreen`. Would you mind moving this in a different place?